### ISSUE TITLE
[Feature Request] Add Server blacklist/whitelist to FreeEmoji plugin

### ISSUE THREAD

**Issue by hydino2085143** [QbDesu/BetterDiscordAddons#45]
Allow us to choose which servers we would like to allow/disallow taking emoji from.

E.g In two servers, Server A and Server B: both have an emoji :peepo: which looks different from each other. If I do not want to see the suggestion for Server B's emoji, I should be able to blacklist it from appearing in the suggestions menu, and :peepo: should not be replaced by it's link when typing in an unrelated field, e.g Server C, or a DM.

**Comment by QbDesu** [comment]
1. This is for Freemoji not DiscordFreeEmoji.
2. Not having the ability to hide individual or all of a server's emoji is more of an Discord issue and would occur with Nitro emojis as well. So if you want that suggest it here https://support.discord.com/hc/en-us/community/topics
3. Freemoji only affects emoji you would normally not be able to send. If you want the ability better differentiate which ones those are disable the "Remove Grayscale Filter" option in the plugin settings

In short, no this won't be a feature of freemoji.
